dns/checkip/src/opnsense/mvc/app/models/OPNsense/DynDNS/CheckIP_Test.xml

```diff
@@ -0,0 +1,33 @@
+<model>
+   <mount>//OPNsense/DynDNS/CheckIP</mount>
+   <version>1.0.0</version>
+   <description>CheckIP settings - Test</description>
+   <items>
+      <test>
+         <service type="OptionField">
+            <default>null</default>
+            <Required>Y</Required>
+            <OptionValues>
+               <null></null>
+            </OptionValues>
+         </service>
```

Comment:
makes this field any sense?

Reply:
Purpose is for not selecting a service (leave option empty).  It's actually the string "null" not the null value.
Changed the string to "auto_select" to be more clear.